packages/ref-metrics-pmp/src/cmip_ref_metrics_pmp/variability_modes.py

```diff
@@ -15,13 +14,28 @@ class ExtratropicalModesOfVariability_PDO(CommandLineMetric):
     slug = "pmp-extratropical-modes-of-variability-pdo"
 
     data_requirements = (
+        DataRequirement(
+            source_type=SourceDatasetType.obs4MIPs,
+            filters=(
+                FacetFilter(
+                    facets={
+                        "source_id": ("HadISST-1-1",),
+                        "variable_id": ("ts",),
+                    }
+                ),
+            ),
```

Comment:
Due to the new requirement, there is nothing run using pmp any more when you only have the decimated test data. Is that intended? Should we add something to the test data? Or should this be an "or" requirement instead of an "and" requirement?